doc/source/readers.rst

```diff
@@ -368,3 +368,13 @@ Arctica-M N1 HDF5 format reader
 
 .. automodule:: satpy.readers.msu_gsa_l1b
     :noindex:
+
+
+netcdf based readers
+--------------------
+
+ESA Ocean Color CCI L3S reader
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. automodule:: satpy.readers.oceancolorcci_l3_nc
```

Comment:
I think this list of readers will be going away shortly and if necessary will be automatically generated. Please remove this new section. At least, that's my opinion.